tests/PKSim.Tests/IntegrationTests/DatabaseUpdateSpecs.cs

```diff
@@ -88,23 +88,22 @@ public void should_adjust_hill_kinetics()
          var rateFormulaRepository = IoC.Resolve<IRateFormulaRepository>();
 
          var allHillFormulas = rateFormulaRepository.All().Where(r => r.Rate.EndsWith("_Hill")).ToArray();
-         allHillFormulas.Count().ShouldBeEqualTo(13);
```

Comment:
@Yuri05 I removed this because this is not accurate anymore and I am not quite sure what this is testing